solr/CHANGES.txt

```diff
@@ -54,7 +54,7 @@ Optimizations
 
 Bug Fixes
 ---------------------
-(No changes)
+* SOLR-16667: Fixed dense/sparse representation in LTR module. (Anna Ruggero)
```

Comment:
this will likely go into 9.9, please change it (we can also do it just before the merge).
If I merge it I'll also add 'via Alessandro Benedetti' that indicates the committer that reviewed and merged